api/src/main/java/org/openmrs/module/fhir2/api/impl/BaseFhirService.java

```diff
@@ -48,7 +48,7 @@ public abstract class BaseFhirService<T extends IAnyResource, U extends OpenmrsO
 	
 	@Autowired
 	@Qualifier("fhirR4")
-	private FhirContext fhirContext;
+	protected FhirContext fhirContext;
```

Comment:
```suggestion
	@Getter(AccessLevel.Protected)
	private FhirContext fhirContext;
```

Then in a subclass, call `getFhirContext()`.

Reply:
i didn't actually need this change after removing the update() and patch() from `observationServiceImpl`